metrics/src/lib.rs

```diff
@@ -324,12 +324,12 @@ pub fn recorder() -> &'static dyn Recorder {
 
 #[doc(hidden)]
 pub fn __private_api_record_count(key: Key, value: u64) {
```

Comment:
`__private_api_record_count` -> `__private_api_increment_counter`?

Reply:
missed that